tools/benchmark_tool/README.md

```diff
@@ -217,78 +217,124 @@ This section provides step-by-step instructions on how to run the Benchmark Tool
+[ INFO ] Network batch size: 1
+[Step 6/11] Configuring input of the model
+[ INFO ] Model input 'data' precision u8, dimensions ([N,C,H,W]): 1 3 224 224
+[ INFO ] Model output 'prob' precision f32, dimensions ([...]): 1 1000
+[Step 7/11] Loading the model to the device
+[ INFO ] Compile model took 109.73 ms
+[Step 8/11] Querying optimal runtime parameters
+[ INFO ] DEVICE: CPU
+[ INFO ]   AVAILABLE_DEVICES  , ['']
+[ INFO ]   RANGE_FOR_ASYNC_INFER_REQUESTS  , (1, 1, 1)
+[ INFO ]   RANGE_FOR_STREAMS  , (1, 8)
+[ INFO ]   FULL_DEVICE_NAME  , 11th Gen Intel(R) Core(TM) i7-11370H @ 3.30GHz
+[ INFO ]   OPTIMIZATION_CAPABILITIES  , ['WINOGRAD', 'FP32', 'FP16', 'INT8', 'BIN', 'EXPORT_IMPORT']
+[ INFO ]   CACHE_DIR  , 
+[ INFO ]   NUM_STREAMS  , 1
+[ INFO ]   AFFINITY  , Affinity.CORE
+[ INFO ]   INFERENCE_NUM_THREADS  , 0
+[ INFO ]   PERF_COUNT  , False
+[ INFO ]   INFERENCE_PRECISION_HINT  , <Type: 'float32'>
+[ INFO ]   PERFORMANCE_HINT  , PerformanceMode.LATENCY
+[ INFO ]   PERFORMANCE_HINT_NUM_REQUESTS  , 0
+[Step 9/11] Creating infer requests and preparing input data
+[ INFO ] Create 1 infer requests took 0.07 ms
+[ WARNING ] No input files were given for input 'data'!. This input will be filled with random values!
+[ INFO ] Fill input 'data' with random values 
+[Step 10/11] Measuring performance (Start inference asynchronously, 1 inference requests, inference only: True, limits: 60000 ms duration)
+[ INFO ] Benchmarking in inference only mode (inputs filling are not included in measurement loop).
+[ INFO ] First inference took 8.74 ms
+[Step 11/11] Dumping statistics report
+Count:          5307 iterations
+Duration:       60015.07 ms
+Latency:
+    Median:     11.19 ms
+    AVG:        11.24 ms
+    MIN:        8.08 ms
+    MAX:        23.97 ms
+Throughput: 88.43 FPS
+```
 
-* For static network:
-   ```
-   [Step 10/11] Measuring performance (Start inference asynchronously, 4 inference requests using 4 streams for CPU, limits: 60000 ms duration)
-   [ INFO ] BENCHMARK IS IN INFERENCE ONLY MODE.
-   [ INFO ] Input blobs will be filled once before performance measurements.
-   [ INFO ] First inference took 26.26 ms
-   Progress: [................... ]  99% done
-
-   [Step 11/11] Dumping statistics report
-   [ INFO ] Count:      6640 iterations
-   [ INFO ] Duration:   60039.70 ms
-   [ INFO ] Latency:
-   [ INFO ]        Median:  35.36 ms
-   [ INFO ]        Avg:    36.12 ms
-   [ INFO ]        Min:    18.55 ms
-   [ INFO ]        Max:    88.96 ms
-   [ INFO ] Throughput: 110.59 FPS
-   ```
+The Benchmark Tool can also be used with dynamically shaped networks to measure expected inference time for various input data shapes. See the -shape and -data_shape argument descriptions in the [All configuration options](#all-configuration-options) section to learn more about using dynamic shapes. Here is a command example for using benchmark_app with dynamic networks and a portion of the resulting output:
```

Comment:
The cross-reference here could break. Need to add an anchor using the <a> tags and use the <a> tags here.

Reply:
Okay, I added the anchor tags!